Fastfile

```diff
@@ -4,77 +4,178 @@ default_platform(:ios)
 
 platform :ios do
 
-  shared_env_variables = nil
-
-  tryouts_prod_release = nil
-  tryouts_preprod_release = nil
-  tryouts_staging_release = nil
+   env_variables = {
+    core: {
+      fastlane_user: "FASTLANE_USER",
+      team_id: "TEAM_ID",
+      workspace: "WORKSPACE",
+      app_name: "APP_NAME",
+      slack_webhook_url: "SLACK_WEBHOOK_URL",
+      #Store
+      app_id: "APP_ID",
+      scheme: "SCHEME",
+      ipa_name: "IPA_NAME",
+      info_plist: "GOOGLE_SERVICE_INFO_PLIST_PATH",
+      app_store_build_config: "APP_STORE_BUILD_CONFIGURATION",
+      itc_name: "FASTLANE_ITC_TEAM_NAME",
+      #Prod
+      prod_scheme: "PROD_SCHEME",
+      prod_ipa_name: "PROD_IPA_NAME",
+      prod_tryouts_app_id: "PROD_TRYOUTS_APP_ID",
+      prod_tryouts_api_token: "PROD_TRYOUTS_API_TOKEN",
+      prod_info_plist: "PROD_GOOGLE_SERVICE_INFO_PLIST_PATH",
+      adhoc_build_config: "ADHOC_BUILD_CONFIGURATION",
+      #Preprod
+      preprod_app_id: "PREPROD_APP_ID", 
+      preprod_scheme: "PREPROD_SCHEME",
+      preprod_ipa_name: "PREPROD_IPA_NAME",
+      preprod_tryouts_app_id: "PREPROD_TRYOUTS_APP_ID",
+      preprod_tryouts_api_token: "PREPROD_TRYOUTS_API_TOKEN",
+      preprod_info_plist: "PREPROD_GOOGLE_SERVICE_INFO_PLIST_PATH",
+      #Staging
+      staging_app_id: "STAGING_APP_ID", 
+      staging_scheme: "STAGING_SCHEME",
+      staging_ipa_name: "STAGING_IPA_NAME",
+      staging_tryouts_app_id: "STAGING_TRYOUTS_APP_ID",
+      staging_tryouts_api_token: "STAGING_TRYOUTS_API_TOKEN",
+      staging_info_plist: "STAGING_GOOGLE_SERVICE_INFO_PLIST_PATH"
+    },
```

Comment:
Let's group them as it is in `required_env_variables`.

Reply:
I grouped them in `env_variables` because if I group them in `required_env_variables`, it cannot linked in same dictionary.